packages/medusa/src/services/shipping-option.ts

```diff
@@ -335,6 +366,73 @@ class ShippingOptionService extends TransactionBaseService {
     })
   }
 
+  async createShippingMethods(
```

Comment:
comment: Bulk create shipping methods. Identical to `createShippingMethod` - except fulfillment validation has moved to the workflow layer.